### PR TITLE
public speaking intro

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date: 2017-03-08T00:39:55-04:00
+---
+
+Public speaking pushed me out of a lot of my comfort zones. I got to learn not only about the content that I was presenting and about the delivery, but also about myself. I found out how I react in certain situations, what feels genuine and what doesn’t, and what I should work on more.
+<br/>
+
+> *Public speaking is a journey of discovering yourself.*
+
+<br/>
+For example, I learned that I can control my speech rate and can use that to improve the effect of my words. But most of all, I learned that indeed, public speaking is one of humanity’s biggest fears, but that I can restrain it and decrease it and that hopefully, one day, I can transform it into contagious energy and enthusiasm for my audience.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: "...and can use that to improve the effect of my words." -> "and how I can use that to improve the effect of my words."

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Public speaking allowed me to get to know myself, what works for me and what not. Here's what I learned about intros.
+tags: [Public speaking, Speaking]
+image:
+date: 2017-03-08T00:39:55-04:00
+---
+
+Public speaking pushed me out of a lot of my comfort zones. I got to learn not only about the content that I was presenting and about the delivery, but also about myself. I found out how I react in certain situations, what feels genuine and what doesn’t, and what I should work on more.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: "about the content that I" -> "about the content I"

File: _posts/blog/2017-03-09-public_speaking_intros.md (hunk 1 above)
Comment (rbordo) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: "But most of all, I learned that indeed, public speaking is one of humanity’s biggest fears, but that I can restrain it and decrease it and that hopefully, one day, I can transform it into contagious energy and enthusiasm for my audience." 

This feels a little bit as "but that I can restrain it..." references to public speaking and not to the biggest fear.

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+<br/>
+
+> *Public speaking is a journey of discovering yourself.*
+
+<br/>
+For example, I learned that I can control my speech rate and can use that to improve the effect of my words. But most of all, I learned that indeed, public speaking is one of humanity’s biggest fears, but that I can restrain it and decrease it and that hopefully, one day, I can transform it into contagious energy and enthusiasm for my audience. 
+
+I saw that before my talk, doing breathing exercises and a power pose stabilises my heart rate and reduces my nervousness. But, once I start talking, I still need a few minutes until I’m calm, and get in my speaking flow. My voice still shakes a bit.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: Maybe: "Doing breathing exercises and power posing before my talk stabilises my..."

Comment (rbordo) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: "get into my speaking flow"?

Comment (rbordo) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: "Before, my voice still shakes a bit."?

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+The <a href="https://www.ted.com/read/ted-talks-the-official-ted-guide-to-public-speaking" target="blank">public speaking theory</a> says that the intro and the conclusion should be the most powerful parts of a talk. I’m far from mastering the conclusion, but I’ve been working on the intro. 
+
+I decided to embrace the emotions and share them with my audience, to use them as a way of creating a better connection with them.
+
+The theme of my latest talk was *"Telling Our Story"*. I started by telling the audience that it’s hard for me to share a personal story in front of strangers and that I want to get to know them a bit first. Then I said my name and what I do and I asked who else in the audience is like me, a developer, who is a QA, who’s a Product Manager and so on, trying to find out who my audience is. Once I knew them a bit, so we weren’t strangers anymore, I declared that I was ready to start sharing my story.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: "Then I said my name..." is a pretty long sentence. ;-)

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+#### This intro had several roles:
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: This intro served several purposes:

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* **It allows the audience to identify with the speaker** — for most of us is hard to share personal things on stage.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: ...for most of us it is hard to share...

personal things -> personal details

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+* It allows me, the speaker, to **get to know the audience** and then, later on, in the intro but also in the conclusion, to mention that what I’m saying is addressed to them; to my fellow devs, to the QAs and PMs in the room. Like this, I’m trying to make the communication feel special for every one of them. To express the fact that **each of them is important to me and are the target of my message**.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: "Like this, ..." -> "This is how I'm trying..."

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* It gives me those two minutes that I need for my voice to **calm down**. 
+
+Of course, the intro differs from one talk to another, depending on the topic, and the target audience. But the main roles for me, are to create a sort of a connection with the audience and to give me those few minutes to get into my flow.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: no comma between "for me are to create"

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+* It allows me, the speaker, to **get to know the audience** and then, later on, in the intro but also in the conclusion, to mention that what I’m saying is addressed to them; to my fellow devs, to the QAs and PMs in the room. Like this, I’m trying to make the communication feel special for every one of them. To express the fact that **each of them is important to me and are the target of my message**.
+* It gives me those two minutes that I need for my voice to **calm down**. 
+
+Of course, the intro differs from one talk to another, depending on the topic, and the target audience. But the main roles for me, are to create a sort of a connection with the audience and to give me those few minutes to get into my flow. 
+
+It took me quite a few talks with crooked beginnings to find what works for me. So don’t be afraid of talking, of experimenting and finding out what works best for you and what doesn’t because being genuine is important.
+
+---
+
+How are your intros? How do you deal with that initial nervousness? How do you try to connect with the audience? I would love to learn more from your experiences.
```
Comment (rbordo) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: ...with your initial nervousness...

File: _posts/blog/2017-03-09-public_speaking_intros.md (hunk 3 above)
Comment (florina-muntenescu) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: ""Before, my voice still shakes a bit."? - it's not "before" but rather in those initial few minutes, my voice still shakes.

File: _posts/blog/2017-03-09-public_speaking_intros.md (hunk 4 above)
Comment (florina-muntenescu) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: I know, but I don't know how to split it better.

Comment (ghost) [reply _posts/blog/2017-03-09-public_speaking_intros.md]: "I introduced myself and asked who else in the room is a developer ...."

File: _posts/blog/2017-03-09-public_speaking_intros.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+The <a href="https://www.ted.com/read/ted-talks-the-official-ted-guide-to-public-speaking" target="blank">public speaking theory</a> says that the intro and the conclusion should be the most powerful parts of a talk. I’m far from mastering the conclusion, but I’ve been working on the intro. 
+
+I decided to embrace the emotions and share them with my audience, to use them as a way of creating a better connection with them.
+
+The theme of my latest talk was *"Telling Our Story"*. I started by telling the audience that it’s hard for me to share a personal story in front of strangers and that I want to get to know them a bit first. Then I said my name and what I do and I asked who else in the audience is like me, a developer, who is a QA, who’s a Product Manager and so on, trying to find out who my audience is. Once I knew them a bit, so we weren’t strangers anymore, I declared that I was ready to start sharing my story.
+
+#### This intro served several purposes:
+* **It allows the audience to identify with the speaker** — for most of us is hard to share personal details on stage.
```
Comment (ghost) [inline _posts/blog/2017-03-09-public_speaking_intros.md]: for most of us *it* is ...

**Comment by ghost** [comment]
👍

**Pull request merged**